### PR TITLE
website: update to the Community page

### PR DESCRIPTION
**Pull request by yy2so**
#### Check the PR title.

- [ ] This PR title match the format: \<type\>(optional scope): \<description\>
- [ ] The description of this PR title is user-oriented and clear enough for others to understand.

#### (Optional) More detailed description for this PR(en: English/zh: Chinese).

en:
zh(optional):

### CODE REVIEW

**Comment by GuangmingLuo** [comment]
There are some typos in the PR, please fix it.

**Comment by yy2so** [comment]
> There are some typos in the PR, please fix it.

Cannot see them, you can point them out if you do, thanks for the review @GuangmingLuo

File: content/en/community/_index.html [hunk 1]
```diff
@@ -39,10 +39,62 @@ <h2>Develop and Contribute</h2>
...
         <i class="fab fa-slack"></i> Slack:
       </a>
-      Join our CloudWeGo Slack Channel
+      Join us on Slack
     </li>
   </ul>
   <p>
-    You can find out how to contribute to these docs in our <a href="https://github.com/cloudwego/kitex/blob/develop/CONTRIBUTING.md">Contribution Guidelines</a>.
-  </p>
+    If you want to become a Contributor, please read the <a href="https://github.com/cloudwego/community/blob/main/CONTRIBUTING.md">Contribution Guidelines</a> and the <a href="https://github.com/cloudwego/community/blob/main/COMMUNITY_MEMBERSHIP.md">Community Membership Priviliges and Guidelines</a>.
```
Comment (GuangmingLuo) [inline content/en/community/_index.html:46]: "Priviliges" should be "Privileges".

File: content/en/community/_index.html [hunk 2]
```diff
@@ -39,10 +39,62 @@ <h2>Develop and Contribute</h2>
...
+      <tr>
+        <th align='left' style='width:40px;'>Role</th>
+        <th align='left' style='width:40px;'>Responsibilities</th>
+        <th align='left' style='width:40px;'>Requirements</th>
+        <th align='left' style='width:40px;'>Defined by</th>
+      </tr>
+      <tr>
+        <td align='left'>Member</td>
+        <td>Active contributor in the community.</td>
+        <td>Sponsorhips by two approvers or maintainers, and multiple code contributions to the project.</td>
```
Comment (GuangmingLuo) [inline content/en/community/_index.html:71]: "Sponsorhips" should be "Sponsorship".

File: content/en/community/_index.html [hunk 3]
```diff
@@ -39,10 +39,62 @@ <h2>Develop and Contribute</h2>
...
+        <td>GitHub org member</td>
+      </tr>
+      <tr>
+        <td align='left'>Committer</td>
+        <td>Active code contributions and/or issue replies in the subproject.</td>
+        <td>Sponsorhips by two approvers or maintainers and multiple code contributions to the project.</td>
```
Comment (GuangmingLuo) [inline content/en/community/_index.html:77]: "Sponsorhips" should be "Sponsorship".

File: content/en/community/_index.html [hunk 4]
```diff
@@ -39,10 +39,62 @@ <h2>Develop and Contribute</h2>
...
+        <td>Approve contributions received by the project.</td>
+        <td>Highly experienced, active reviewer and contributor to a subproject.</td>
+        <td>GitHub subproject committer team.</td>
+      </tr>
+      <tr>
+        <td align='left'>Maintainer</td>
+        <td>Set the direction and priorities for a subproject.</td>
+        <td>Demonstrated sense of responsibility and excellent technical judgement for the subproject.</td>
+        <td>GitHub subproject committer team</td>
+      </tr>
```
Comment (GuangmingLuo) [inline content/en/community/_index.html:97]: Those are not correct. Please be more careful when do the copying from other places. 
Reviewer -> GitHub subproject reviewer Team
Approver -> GitHub subproject approver Team
Maintainer -> GitHub subproject maintainer Team

Comment (GuangmingLuo) [reply content/en/community/_index.html:97]: refer : https://github.com/cloudwego/community/blob/main/COMMUNITY_MEMBERSHIP.md

**Comment by yy2so** [comment]
Thanks, appreciate the review @GuangmingLuo, updated

**Comment by rogerogers** [comment]
image
The old style needs to be modified accordingly, if there is a need, I can make the final edits.

**Comment by yy2so** [comment]
> image The old style needs to be modified accordingly, if there is a need, I can make the final edits.

Thank you @rogerogers That would be very much appreciated

**Review by GuangmingLuo — APPROVED** [review]

**Pull request merged**